src/commands/auth/use.ts

```diff
@@ -0,0 +1,173 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import * as inquirer from 'inquirer'
+
+import { CheApiClient } from '../../api/che-api-client'
+import { CheServerLoginManager } from '../../api/che-login-manager'
+import { CHE_API_ENDPOINT_KEY, username, USERNAME_KEY } from '../../common-flags'
+
+export default class Use extends Command {
+  static description = 'Set active login session'
+
+  static args = [
+    {
+      name: CHE_API_ENDPOINT_KEY,
+      description: 'Eclipse Che server API endpoint',
+      env: 'CHE_API_ENDPOINT',
+      required: false
+    }
+  ]
+  static flags: flags.Input<any> = {
+    help: flags.help({ char: 'h' }),
+    [USERNAME_KEY]: username,
+    interactive: flags.boolean({
+      char: 'i',
+      description: 'Select active login session in interactive mode',
+      required: false,
+      exclusive: [USERNAME_KEY]
+    }),
+  }
+
+  static examples = [
+    '# Make given user on specified cluster current:\n' +
+    'auth:use che-che.apps-crc.testing/api -u username',
+    '# Switch to another user on the same cluster:\n' +
```

Comment:
Set an active login session for the specified user on the current cluster

Reply:
I think current version is clearer.